people.php

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<?php
+	$conn = require_once (__DIR__ . "/scripts/connection.php");
+	if (isset($_GET['id'])) {
+		$id = $_GET['id'];
+	} else {
+		# echo ("no person selected");
+		echo "
+					<title>People</title>
+					<link rel='icon' href='./img/default_icon.png'>
+					<link rel='stylesheet' href='./css/fonts.css'>
+					<link rel='stylesheet' href='/css/menu_people.css'>
+					";
+		require (__DIR__ . "/scripts/menus.php");
```

Comment:
```suggestion
		require_once (__DIR__ . "/scripts/menus.php");
```
file defines a function